### PR TITLE
#1341 added my name to the credits page.

### LINKED ISSUE #1341: Quinn Louie - Credits
### Description

Giving myself credit for the amazing developing work I will be doing at NER!

### Acceptance Criteria

See my name in my favorite color on the credits page.

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by QuinnJL04**
## Changes

Added my name to credits.

## Notes

I kind of struggled as I have minimal experience on github. I ran into some issues in the pull request as it was not giving me the option to make one when I would select my issue. Eventually I got it to work, but I am not sure if that caused me to make a pull request incorrectly.

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window.

Screenshot 2023-09-13 202950

Screenshot 2023-09-13 202925

Closes #1341

### CODE REVIEW

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -85,6 +85,7 @@ const CreditsPage: React.FC = () => {
     { name: 'Liam Kosar', color: '#eb66ff' },
     { name: 'Daniel Yu', color: '#bdc0c7' },
     { name: 'William Seward', color: '#e53774' },
+    { name: 'Quinn Louie', color: '#3281a8'},
```
Comment (walker-sean) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:88]: Can you add a space after the closing ' to match the other lines?

**Review by walker-sean — DISMISSED** [review]
Great job! Just a tiny thing to fix so that the Prettier (code format) test passes!

File: .env
Comment (ninest) [inline .env:1]: Can you remove this file from the PR?

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**